src/Paper.js

```diff
@@ -385,7 +385,7 @@ class Paper {
 			const slugify = researcher.getResearch( 'slugify' )
 			const removePunctuation = researcher.getResearch( 'removePunctuation' )
 			const keywordLower = this.getLower( 'keyword' )
-				.split( '.' ).join( '' )
+				.split( '.' ).join( '-' )
```

Comment:
What is the reason for using `split()` and then `join()`? Why don't we do `.replace(/\./g, '-')`?